app/services/to_cocina/collection_mapper.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module ToCocina
+  # Maps CollectionForm to Cocina Collection
+  class CollectionMapper
+    def self.call(...)
+      new(...).call
+    end
+
+    # @param [CollectionForm] collection_form
+    # @param [source_id] source_id
+    def initialize(collection_form:, source_id:)
+      @collection_form = collection_form
+      @content = content
+      @source_id = source_id
```

Comment:
Do collections have source ids in H2?

Reply:
Hmm, I don't think they do looking at them. Will remove.

Reply:
I stand corrected, at least this one does have a source id:

https://argo.stanford.edu/view/druid:cm517jf6980